### PR TITLE
add Foreman 1.24 config

### PR DESCRIPTION
**Pull request by zjhuntin**

### CODE REVIEW

**Comment by pcreech** [comment]
The config looks good to me, bu would like an additional ack from someone else on that part.

Also, maybe we should also add the mash_scripts/1.24.0/*.mash files in this PR?

**Comment by zjhuntin** [comment]
@pcreech added

File: mash_scripts/foreman/1.24.0/foreman-client-1.24-fedora28.mash [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[foreman-client-1.24-fedora28]
```
Comment (ekohl) [inline mash_scripts/foreman/1.24.0/foreman-client-1.24-fedora28.mash]: F28 is EOL. Do we really want to build packages for this? I think we also dropped it in nightly configs some time ago.

Comment (zjhuntin) [reply mash_scripts/foreman/1.24.0/foreman-client-1.24-fedora28.mash]: ACK, i'll remove it

File: configs/foreman/1.24.yaml [hunk 2]
```diff
@@ -0,0 +1,536 @@
...
+  :1.24.0:
+    :redmine_version_id: 1070
+:mash_scripts:
+  - foreman-1.24-rhel7-dist
+  - foreman-plugins-1.24-rhel7-dist
+  - foreman-rails-1.24-rhel7
+  - foreman-client-1.24-rhel7
+  - foreman-client-1.24-rhel6
+  - foreman-client-1.24-rhel5
+  - foreman-client-1.24-fedora28
```
Comment (ekohl) [inline configs/foreman/1.24.yaml]: This should be dropped

File: configs/foreman/1.24.yaml [hunk 3]
```diff
@@ -0,0 +1,536 @@
...
+  - foreman-client-1.24-fedora29
+  - foreman-client-1.24-sles11
+  - foreman-client-1.24-sles12
+:repos: []
+:gpg_key: 7B5B366A
```
Comment (ekohl) [inline configs/foreman/1.24.yaml]: This is the 1.23 GPG key

**Pull request merged**